pages/cookies.tsx

```diff
@@ -0,0 +1,190 @@
+      JSON.parse(localStorage.getItem('hotjarCookiesAccepted') || 'false')
+    )
+  }, [])
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target
+    setCookiesAccepted(JSON.parse(value))
+  }
+
+  const handleSubmit = (e: React.SyntheticEvent) => {
+    e.preventDefault()
+    localStorage.setItem(
+      'hotjarCookiesAccepted',
+      JSON.stringify(cookiesAccepted)
+    )
+  }
+
+  return (
+    <Layout>
+      <HeaderComponent title="Support in Lambeth" />
+      <MainBody>
+        <Heading>Cookies</Heading>
+        <Text>
+          Cookies are files saved on your phone, tablet or computer when you
+          visit a website.
+        </Text>
+        <Text>
+          We use cookies to store information about how you use this site, such
+          as the pages you visit.
+        </Text>
+        <Heading>Cookie Settings</Heading>
+        <Text>
+          We use 2 types of cookies. You can choose which cookies you are happy
+          for us to use.
+        </Text>
+        <SubHeading>
+          Hotjar cookies that measure website use and collect feedback
+        </SubHeading>
+        <Text>
+          We use Hotjar Analytics to measure how you use the website to collect
+          feedback from you about our website so we can improve it based on user
+          needs.
+        </Text>
+        <Text>
+          Hotjar sets cookies that store anonymised information about:
+        </Text>
+        <ul>
+          <li>how you got to the site</li>
+          <li>the pages you visit on and how long you spend on each page</li>
+          <li>what you click on while you are visiting the site</li>
+          <li>whether you have already visited the site</li>
+        </ul>
+        <Text>
+          For further details please see the about Hotjar section of{' '}
+          <a href="https://help.hotjar.com/hc/en-us/categories/115001323967-About-Hotjar">
+            Hotjar support site
+          </a>
+          .
+        </Text>
+        <form id="hotjarCookies" onSubmit={handleSubmit}>
+          <div>
+            <label>
+              <input
+                onChange={handleChange}
+                type="radio"
+                id="hotjarCookiesAccept"
+                name="hotjarCookiesAccepted"
+                value="true"
+                checked={cookiesAccepted}
+              />
+              Use Hotjar cookies
+            </label>
+            <label>
+              <input
+                onChange={handleChange}
+                type="radio"
+                id="hotjarCookiesReject"
+                name="hotjarCookiesAccepted"
+                value="false"
+                checked={!cookiesAccepted}
+              />
+              Do not use Hotjar cookies
+            </label>
+          </div>
+          <SubHeading>Strictly necessary cookies</SubHeading>
+          <Text>
+            We use essential cookies to remember your cookie preferences.
```

Comment:
We don't use a cookie for this, we use local storage 🙂

Reply:
Local storage is considered a "cookie like" technology that we have to tell people about according to ico guidance. (https://ico.org.uk/for-organisations/guide-to-pecr/guidance-on-the-use-of-cookies-and-similar-technologies/what-are-cookies-and-similar-technologies/)

Reply:
We may have to say we use it for the quiz now I come to think about it